src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1159,22 +1165,27 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 				$body[] = __( 'These plugins are now up to date:' );
 
 				foreach ( $successful_updates['plugin'] as $item ) {
+					$body_message = '';
 					if ( $item->item->current_version ) {
-						$body[] = sprintf(
+						$body_message .= sprintf(
 							/* translators: 1: Plugin name, 2: Current version number, 3: New version number. */
 							__( '- %1$s (from version %2$s to %3$s)' ),
 							$item->name,
 							$item->item->current_version,
 							$item->item->new_version
 						);
 					} else {
-						$body[] = sprintf(
+						$body_message .= sprintf(
 							/* translators: 1: Plugin name, 2: Version number. */
 							__( '- %1$s version %2$s' ),
 							$item->name,
 							$item->item->new_version
 						);
 					}
+					if ( $item->item->url ) {
```

Comment:
```suggestion
					if ( ! empty( $item->item->url ) ) {
```

Safety check: WordPress.org returns this but it may not be included in some commercial plugins' endpoints as they haven't needed to in the past.